### DISCUSSION TITLE
Maya file export selected should not export upstream nodes

### DISCUSSION THREAD

**Discussion by sharktacos** [Autodesk/maya-usd#3292] [Ideas] [answered]
_File > Export selected_ will export from the root of the scene upstream of the selection.  I would expect it to instead export the selected node and its children. This can be done with the **mayaUSDExport** -exportRoots option (setting it to the selected DAG node).

I'm proposing for discussion that this should be the default behavior for the menu "_File > Export selected_"

**Comment by wallworm** [comment, marked as answer]
Thanks for your suggestion. I'll bring this up with the team.
